flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/expressions/call.scala

```diff
@@ -61,3 +70,11 @@ object BuiltInFunctionConstants {
   val TRIM_TRAILING = Literal(2)
   val TRIM_DEFAULT_CHAR = Literal(" ")
 }
+
+object BuiltInMethods {
```

Comment:
I wouldn't place the BuiltInMethods here because we then mix user-facing API and runtime. Can you put it either into `MethodCallGenerator` or into its package?